server/engine/src/main/java/org/eclipse/lsp/cobol/dialects/ibm/TransformTreeStage.java

```diff
@@ -258,8 +258,9 @@ private void registerProcessors(AnalysisConfig analysisConfig, ProcessingContext
     // Phase USAGE
```

Comment:
This whole change still looks overly complicated. As far as I can tell, the whole problem could be solved by introducing a new processor (e.g. in the post definition phase) that would transform table call nodes into function reference nodes when the table call "variable" name referes to a function in the repository.